### PR TITLE
fix: adjustment of the color of chevron on selection

### PR DESCRIPTION
**Pull request by emrecevik06**
There was a problem with the color of the chevron when selecting in the light theme. When selecting, the chevron was still dark and did not use the color of the text (white).

### CODE REVIEW

File: packages/control-property-editor/src/panels/outline/Tree.tsx [hunk 1]
```diff
@@ -413,7 +413,7 @@ export const Tree = (): ReactElement => {
                     onContextMenu={(e) => isExtensionPoint && handleOpenTooltip(e, tooltipId)}>
                     {groupHeaderProps?.group?.count !== 0 && (
                         <Icon
-                            className={`${chevronTransform}`}
+                            className={`${chevronTransform} chevron`}
```
Comment (Jimmy-Joseph19) [inline packages/control-property-editor/src/panels/outline/Tree.tsx:416]:
```suggestion
                            className={`${chevronTransform} active-foreground`}
```

**Review by Jimmy-Joseph19 — APPROVED** [review]
Changes looks good.
Changeset found
Approving

File: packages/control-property-editor/src/panels/outline/Tree.tsx [hunk 2]
```diff
@@ -413,7 +413,7 @@ export const Tree = (): ReactElement => {
                     onContextMenu={(e) => isExtensionPoint && handleOpenTooltip(e, tooltipId)}>
                     {groupHeaderProps?.group?.count !== 0 && (
                         <Icon
-                            className={`${chevronTransform}`}
+                            className={`${chevronTransform} active-foreground`}
```
Comment (voicis) [inline packages/control-property-editor/src/panels/outline/Tree.tsx:416]: I don't think we should add new class for this. We already have a class for selected row that we can use `app-panel-selected-bg` in combination with the row class `tree-row`. All icons in the tree should be handled generically and we shouldn't need to fix  this for each of them, but with current solution extension icons when selected still has wrong colour.

image

**Review by voicis — CHANGES_REQUESTED** [review]

**Review by voicis — APPROVED** [review]
- Straight forward style change
- Review comments addressed
- Changeset present
- Tested locally

**Review by Jimmy-Joseph19 — APPROVED** [review]
Feedback Changes looks good
Approving

**Pull request merged**